portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase

```diff
@@ -0,0 +1,235 @@
+			entryTitle = "Blogs Entry Title");
+
+		Button.clickPublish();
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Test Site Name");
+
+		task ("And a 2 Blog Entries exists") {
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "My Content",
+				entryTitle = "My Blogs");
+		}
+
+		task ("Then Blog Entry 1 should not exist") {
+			BlogsNavigator.openToAddEntry(
+				layoutName = "Blogs Page",
+				siteURLKey = "test-site-name");
+
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Test Site Name");
+
+			BlogsEntry.moveToRecycleBinPG(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title");
+		}
+
+		task ("And Blog Entry 2 should exist") {
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Test Site Name");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "My Content",
+				entryTitle = "My Blogs");
+		}
+	}
+
+	@description = "Can Edit Blog"
+	@priority = 5
+	test CanEditBlog {
+		task ("And a Blog Entry exists") {
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Test Site Name");
+
+			BlogsNavigator.openToAddEntry(
+				layoutName = "Blogs Page",
+				siteURLKey = "test-site-name");
+
+			BlogsEntry.addEntryContent(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title");
+
+			Button.clickPublish();
+
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Test Site Name");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title");
+		}
+
+		task ("When the User edits the blog Entry") {
+			BlogsNavigator.gotoEditPG(entryTitle = "Blogs Entry Title");
+
+			BlogsEntry.editTitle(
+				entryTitle = "Blogs Entry Title",
+				entryTitleEdit = "Blogs Entry Title 1");
+
+			BlogsEntry.editContent(
+				entryContent = "Blogs Entry Content",
+				entryContentEdit = "Blogs Entry Content 1");
+
+			Button.clickPublish();
+		}
+
+		task ("Then the Blog Entry should be edited") {
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blogs Entry Content 1",
+				entryTitle = "Blogs Entry Title 1");
+		}
+	}
+
+	@description = "Can Search Blogs"
+	@priority = 5
+	test CanSearchBlogs {
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "test-site-name");
+
+		BlogsEntry.addEntryContent(
+			entryContent = "My Content",
+			entryTitle = "My Blogs");
+
+		Button.clickPublish();
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Test Site Name");
+
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "test-site-name");
+
+		BlogsEntry.addEntryContent(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry Title");
+
+		Button.clickPublish();
+
+		Navigator.gotoSitePage(
+			pageName = "Blogs Page",
+			siteName = "Test Site Name");
```

Comment:
this steps should be replaced by a simple json macro to add a second blog entry since it's not the main goal of this test.
Don't forget adding a task for this step.